app/views/admin/documentations/index.html.slim

```diff
@@ -0,0 +1,16 @@
+- content_for(:title, t('documentation.index'))
+
+- @documentations.each do |calling_controller, documentation_list|
+  h2= t("controllers.admin.#{calling_controller}")
+  - documentation_list.each do |documentation|
+    table.table
+      thead
+        tr
+          td= documentation_title(documentation)
+          td= documentation_location(documentation)
+          td= documentation_division(documentation)
+          td.text-right= documentation_edit(documentation)
```

Comment:
Do we need to check here if the user has permission to edit before showing the edit link? in QA, when I clicked to edit, I was allowed to get to the edit page and use the form, but when i clicked submit button ("Update Documentation") I was redirected to https://ms.tww.org/users/sign_in . ..

Reply:
update - issue is with legacy redirect based on url saved to db. For now, hide edit button; change redirect behavior in issue next sprint